tools/hsmtool.c

```diff
@@ -417,6 +419,25 @@ static int dumponchaindescriptors(const char *hsm_secret_path, const char *passw
 
 	wally_free_string(enc_xpub);
 
+	/* If they also want xpriv break a line and output them as well. */
+	if (private) {
+		printf("\n");
+		if (bip32_key_to_base58(&master_extkey, BIP32_FLAG_KEY_PRIVATE, &enc_xpub) != WALLY_OK)
+			errx(ERROR_LIBWALLY, "Can't encode xpub");
+
+		descriptor = tal_fmt(NULL, "wpkh(%s/0/0/*)", enc_xpub);
+		descriptor_checksum(descriptor, strlen(descriptor), checksum, sizeof(checksum));
+		printf("%s#%s\n", descriptor, checksum);
+		tal_free(descriptor);
```

Comment:
this is the 3rd time you're repeating this code (and you do it again below).

Reply:
This commit is now removed.